tests/unit_tests/scheduler/test_openpbs_driver.py

```diff
@@ -203,20 +203,22 @@ async def test_cluster_label():
     "Job id            Name             User              Time Use S Queue\n"
     "----------------  ---------------- ----------------  -------- - -----\n"
 )
+"Job id            Name             User              Time Use S Queue\n----------------  ---------------- ----------------  -------- - -----"
```

Comment:
The header on line 203 and 204 is the short header, isn't it? Should we include that in the test or should we skip it?

Reply:
I dont think it matters as it is being skipped anyways.